packages/fxa-auth-server/lib/routes/session.js

```diff
@@ -692,6 +693,92 @@ module.exports = function (
         return {};
       },
     },
+    {
+      method: 'GET',
+      path: '/session/verify/token',
+      options: {
+        ...SESSION_DOCS.SESSION_VERIFY_TOKEN_GET,
+        auth: {
+          strategy: 'sessionToken',
+        },
+        response: {
+          schema: isA.object({
+            accountStatus: isA
+              .string()
+              // account status cannot be null - if no account exists for the session token's uid
+              // for example if the account has been deleted since the session token was created
+              // the endpoint's authentication strategy will return an "INVALID_TOKEN" error
+              .valid('verified', 'unverified')
+              .required(),
+            sessionStatus: isA
+              .string()
+              .valid('verified', 'mustVerify', 'mustUpgrade', null)
+              .required(),
+            verificationMethod: isA.string().optional(),
```

Comment:
I see we have these set exactly like this in `session/reauth` for example but could we add a `.valid()` here too?